edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_start.html

```diff
@@ -26,7 +26,7 @@
     <div>
       <button type="button" class="proctored-enter-exam" data-action="start" data-exam-id="{{exam_id}}" data-change-state-url="{{change_state_url}}">
         {% blocktrans %}
-          I'm ready! Start my timed proctored exam
+          I'm ready! Start my exam
```

Comment:
Sorry, I wasn't clear in my comment in PR#17. Please remove "I'm ready!" so that the button text reads "Start my exam".